### PR TITLE
deleteTileCache

### PR DESCRIPTION
**Pull request by jread-usgs**
#70

### CODE REVIEW

**Comment by jread-usgs** [comment]
@jiwalker-usgs review if you can

File: NAMESPACE [hunk 1]
```diff
@@ -1,10 +1,13 @@
-# Generated by roxygen2 (4.1.0): do not edit by hand
+# Generated by roxygen2 (4.1.1): do not edit by hand
```
Comment (jiwalker-usgs) [inline NAMESPACE]: This is kind of annoying.

File: man/addLayer.Rd [hunk 2]
```diff
@@ -1,4 +1,4 @@
-% Generated by roxygen2 (4.1.0): do not edit by hand
+% Generated by roxygen2 (4.1.1): do not edit by hand
```
Comment (jiwalker-usgs) [inline man/addLayer.Rd]: This is more where it is annoying.

File: man/authenticateUser.Rd [hunk 3]
```diff
@@ -1,4 +1,4 @@
-% Generated by roxygen2 (4.1.0): do not edit by hand
+% Generated by roxygen2 (4.1.1): do not edit by hand
 % Please edit documentation in R/authenticateUser.R
```
Comment (jread-usgs) [inline man/authenticateUser.Rd]: thought it was here

File: man/checkAuth.Rd [hunk 4]
```diff
@@ -1,4 +1,4 @@
-% Generated by roxygen2 (4.1.0): do not edit by hand
+% Generated by roxygen2 (4.1.1): do not edit by hand
```
Comment (jiwalker-usgs) [inline man/checkAuth.Rd]: And here.

**Comment by jread-usgs** [comment]
@jiwalker-usgs good to merge?

**Comment by jiwalker-usgs** [comment]
I think this should be good, not sure why it is being weird on the service side :shipit:

**Pull request merged**